src/client/dfs/dfs.c

```diff
@@ -7287,3 +7287,12 @@ dfs_obj_fix_type(dfs_t *dfs, dfs_obj_t *parent, const char *name)
 		D_FREE(entry.value);
 	return rc;
 }
+
+mode_t
+dfs_obj_mode(dfs_t *dfs, dfs_obj_t *obj)
```

Comment:
there is already dfs_get_mode. why add this?

Reply:
Sorry. I missed it.
How can we get the mode of dfs root?

Reply:
well you don't even need to use that. dfs_lookup() gives you mode already?
not sure why you need to do get mode in the first place..